resources/tenants/controller/tenants.go

```diff
@@ -18,22 +18,34 @@ import (
 	"net/http"
 
 	"github.com/ant0ine/go-json-rest/rest"
+	deploymentsController "github.com/mendersoftware/deployments/resources/deployments/controller"
+	deploymentsModel "github.com/mendersoftware/deployments/resources/deployments/model"
+	"github.com/mendersoftware/deployments/resources/tenants/model"
+	"github.com/mendersoftware/go-lib-micro/identity"
 	"github.com/mendersoftware/go-lib-micro/log"
 	"github.com/mendersoftware/go-lib-micro/rest_utils"
-
-	"github.com/mendersoftware/deployments/resources/tenants/model"
 )
 
 type Controller struct {
 	model model.Model
 }
 
+type TenantDeploymentController struct {
+	model deploymentsModel.DeploymentsModel
+}
+
 func NewController(model model.Model) *Controller {
 	return &Controller{
 		model: model,
 	}
 }
 
+func NewDeploymentsTenantController(model deploymentsModel.DeploymentsModel) *TenantDeploymentController {
```

Comment:
Why not use existing Controller instead adding another one.

Reply:
It's not possible to create a new method in a non-local package (I cant create a method of deploymentsController inside the 'tenant' package)

Reply:
Controller is in same package, it's in the same file.

Reply:
right, I see what you mean now, and fixed it.